homeassistant/components/volvooncall/__init__.py

```diff
@@ -74,9 +123,26 @@ async def async_setup_entry(
 
     volvo_data = VolvoData(hass, connection, entry)
 
-    hass.data[DOMAIN][entry.entry_id] = VolvoUpdateCoordinator(hass, volvo_data)
+    coordinator = hass.data[DOMAIN][entry.entry_id] = VolvoUpdateCoordinator(
+        hass, volvo_data
+    )
+
+    try:
+        await coordinator.async_config_entry_first_refresh()
+    except ConfigEntryAuthFailed:
+        return False
+    except UpdateFailed:
+        return False
```

Comment:
This won't raise `UpdateFailed`. It will raise `ConfigEntryNotReady` when the first update fails. It means the integration will try to set up again later.
```suggestion
```